src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -11,65 +11,74 @@
 
 #pylint: disable=line-too-long
 
+custom_path = 'azure.cli.command_modules.vm.custom#{}'
+mgmt_path = 'azure.mgmt.compute.operations.{}#{}.{}'
+
 # VM
 
 cli_command(__name__, 'vm create', 'azure.cli.command_modules.vm.mgmt_vm.lib.operations.vm_operations#VmOperations.create_or_update', cf_vm_create,
             transform=DeploymentOutputLongRunningOperation('Starting vm create'))
 
-cli_command(__name__, 'vm delete', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.delete', cf_vm)
-cli_command(__name__, 'vm deallocate', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.deallocate', cf_vm)
-cli_command(__name__, 'vm generalize', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.generalize', cf_vm)
-cli_command(__name__, 'vm show', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.get', cf_vm)
-cli_command(__name__, 'vm list-vm-resize-options', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.list_available_sizes', cf_vm)
-cli_command(__name__, 'vm stop', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.power_off', cf_vm)
-cli_command(__name__, 'vm restart', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.restart', cf_vm)
-cli_command(__name__, 'vm start', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.start', cf_vm)
-cli_command(__name__, 'vm redeploy', 'azure.mgmt.compute.operations.virtual_machines_operations#VirtualMachinesOperations.redeploy', cf_vm)
-cli_command(__name__, 'vm list-ip-addresses', 'azure.cli.command_modules.vm.custom#list_ip_addresses')
-cli_command(__name__, 'vm get-instance-view', 'azure.cli.command_modules.vm.custom#get_instance_view')
-cli_command(__name__, 'vm list', 'azure.cli.command_modules.vm.custom#list_vm')
-cli_command(__name__, 'vm resize', 'azure.cli.command_modules.vm.custom#resize_vm')
-cli_command(__name__, 'vm capture', 'azure.cli.command_modules.vm.custom#capture_vm')
-cli_command(__name__, 'vm open-port', 'azure.cli.command_modules.vm.custom#vm_open_port')
+op_var = 'virtual_machines_operations'
+op_class = 'VirtualMachinesOperations'
+cli_command(__name__, 'vm delete', mgmt_path.format(op_var, op_class, 'delete'), cf_vm)
```

Comment:
//cc @derekbekoe. I am fine with this new style. As long as we agree, let us propagate this style across moving forward